schema/ingestion_config/v1.0.0/codegen/ingestion_config_models.schema.json

```diff
@@ -1,4834 +0,0 @@
-{
```

Comment:
Also, curious about this file being deleted.

Reply:
because of https://github.com/chanzuckerberg/cryoet-data-portal-backend/issues/274. I had to downgrade linkml and regenerate